dev-python/matplotlib/matplotlib-2.2.2-r1.ebuild

```diff
@@ -19,15 +19,15 @@ SLOT="0"
 # Fonts: BitstreamVera, OFL-1.1
 LICENSE="BitstreamVera BSD matplotlib MIT OFL-1.1"
 KEYWORDS="amd64 ~arm arm64 ~ppc ppc64 x86"
-IUSE="cairo doc excel examples gtk3 latex qt5 test tk wxwidgets"
+IUSE="cairo doc excel examples gtk3 latex test tk wxwidgets"
```

Comment:
why is this being dropped? does it require py2.7?

Reply:
It causes py27 backpressure on PyQt5 due to still having python2_7 inside PYTHON_COMPAT.

Reply:
oh I didn't realize this is matplotlib 2.x... ok then I don't care ;)